src/main/java/com/securian/creditcompass/login/LoginInteractor.java

```diff
@@ -0,0 +1,40 @@
+package com.securian.creditcompass.login;
+
+import com.securian.creditcompass.entities.ClaimsExaminer;
+import org.springframework.stereotype.Service;
+
+import javax.naming.AuthenticationException;
+import java.util.Optional;
+
+@Service
+public class LoginInteractor implements LoginInputBoundary{
+    private final LoginDataAccessInterface loginDataAccessInterface;
+    public LoginInteractor(LoginDataAccessInterface loginDataAccessInterface){
+        this.loginDataAccessInterface = loginDataAccessInterface;
+    }
+
+    @Override
+    public boolean authenticate(LoginInputData loginInputData) throws AuthenticationException{
```

Comment:
Probably good to return Boolean instead of the primitive boolean for OOP reasons, I believe the TA's or Maryam said something about this.